exercises/difference-of-squares/HINTS.md

```diff
@@ -1,3 +1,4 @@
 ## Hints
 For this exercise the following F# feature comes in handy:
```

Comment:
Good find! I've updated the PR. By the way, if you merge this, can you do that using the "Squash and merge" option? In general, this is how I merge PR's, as we then end up with one combined commit, instead of several smaller ones. cc @jwood803